meinberlin/apps/plans/migrations/0068_plan_add_point_field.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.16 on 2025-02-19 15:54
+
+import django.contrib.gis.db.models.fields
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("meinberlin_plans", "0067_replace_point_field"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="plan",
+            name="point",
+            field=django.contrib.gis.db.models.fields.PointField(
+                blank=True,
+                null=True,
+                srid=4326,
+                verbose_name="Can your plan be located on the map?",
+            ),
+        ),
+    ]
```

Comment:
so here renaming the field from `geos_point` to `point` it doesn't work?

Reply:
no, same as in a4, that's why we split it into deleting and readding